assets/js/base/components/chip/index.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+/**
+ * Component used to render a "chip" -- a list item containing some text with
+ * an X button to remove/dismiss each chip.
+ *
+ * Each chip defaults to a list element but this can be customized by providing
+ * a wrapperElement.
+ */
+const Chip = ( {
+	text,
+	screenReaderText,
+	element = 'li',
+	className = '',
+	onRemove = () => {},
+	disabled = false,
+	radius = 'small',
+} ) => {
+	const Wrapper = element;
+	const wrapperClassName = classNames(
+		className,
+		'wc-block-components-chip',
+		'wc-block-components-chip--radius-' + radius
+	);
+
+	return (
+		// @ts-ignore
+		<Wrapper className={ wrapperClassName }>
+			<span aria-hidden="true" className="wc-block-components-chip__text">
+				{ text }
+			</span>
+			<span className="screen-reader-text">
+				{ screenReaderText ? screenReaderText : text }
+			</span>
+			<button
+				className="wc-block-components-chip__remove"
+				onClick={ onRemove }
+				disabled={ disabled }
+				aria-label={ sprintf(
+					/* translators: %s chip text. */
+					__( 'Remove "%s"', 'woo-gutenberg-products-block' ),
```

Comment:
When navigating with the screen reader, this is read twice (similar to #1775). I'm not sure if this a bug with the Linux screen reader or we are doing something wrong. But might be good if somebody can double check with another system. In any case, I don't think this is a blocker.

Reply:
No duplication for me with voiceover on chrome.